### DISCUSSION TITLE
How about swapping zod for suretype?

### DISCUSSION THREAD

**Discussion by capaj** [blitz-js/blitz#2913] [Ideas]
I came across this package today:
https://github.com/grantila/suretype

would be worth measuring how faster it can make blitz API if zod is swapped for suretype.
Definetely seems like it could be a lot faster for non trivial shapes of data. In this bench https://github.com/grantila/suretype/blob/master/benchmark/index.ts zod is 2568x slower.
Is there a benchmark in blitz where we could measure this?

**Comment by flybayer** [comment]
You can certainly switch it out in your code if you want. We don't use zod anywhere internally.

Suretype looks nice, just not as full featured as Zod.

**Reply by davidharting** [reply]
Oh interesting! For some some reason I thought the use of SuperJSON internally required Zod.
